manifests/mod/ssl.pp

```diff
@@ -11,6 +11,12 @@
     'freebsd' => '/var/run/ssl_scache(512000)',
   }
 
+  $stapling_cache = $::osfamily ? {
+    'debian'  => "\${APACHE_RUN_DIR}/ocsp(128000)",
+    'redhat'  => '/var/cache/mod_ssl/ocsp(128000)',
+    'freebsd' => '/var/run/ocsp(128000)',
```

Comment:
Is there any reasonable value we can set as a default case? Otherwise this will cause issues with strict_variables.